### PR TITLE
Feature locally mute remote audio

### PR DESCRIPTION
**Pull request by tyleralves**
See #123 
iOS only. Unfortunately my project only uses iOS and my Java skills are not the best.

### CODE REVIEW

File: ios/RCTTWVideoModule.m [hunk 1]
```diff
@@ -101,6 +101,14 @@ - (void)addParticipantView:(TVIVideoView *)view sid:(NSString *)sid trackSid:(NS
   }
 }
 
+RCT_EXPORT_METHOD(muteRemoteAudio:(NSString *)participantSid) {
```
[the remark this answers is not in the document]
Comment (tyleralves) [reply ios/RCTTWVideoModule.m]: Haha, yes. I have a unique situation where I never want audio, but I will add the unmute method to address the majority of users.

**Comment by tyleralves** [comment]
@slycoder This works for enable/ disable with my project. Used like this:

```
onParticipantAddedAudioTrack = ({ participant }) => {
    const { sid: participantSid } = participant;
    this.twilioVideo.setRemoteAudioPlayback({ participantSid, enabled: false });
};
```

File: ios/RCTTWVideoModule.m [hunk 2]
```diff
@@ -101,6 +101,14 @@ - (void)addParticipantView:(TVIVideoView *)view sid:(NSString *)sid trackSid:(NS
   }
 }
 
+RCT_EXPORT_METHOD(setRemoteAudioPlayback:(NSString *)participantSid enabled:(BOOL)enabled) {
+    TVIRemoteParticipant *participant = [self.room getRemoteParticipantWithSid:participantSid];
+    NSArray<TVIRemoteAudioTrackPublication *> *trackPublications = participant.remoteAudioTracks;
```
Comment (slycoder) [inline ios/RCTTWVideoModule.m]: Mind adding a check for a null participant?

Comment (slycoder) [reply ios/RCTTWVideoModule.m]: I just figure you might have a race condition where the track is added, and then they leave the room but in the meantime on the JS side someone is calling setRemoteAudioPlayback via the callback.

**Comment by slycoder** [comment]
Thanks! 👏

**Comment by brunodias28** [comment]
How to do this on android ?

**Comment by Balasnest** [comment]
I appreciate this PR! @tyleralves

**Comment by codemeall** [comment]
do we have this same feature for android as well ?

**Pull request merged**